packages/docs/PropTables/CheckboxPropTable.tsx

```diff
@@ -39,3 +72,21 @@ export const CheckboxPropTable: React.FC<PropTableWrapper> = props => (
     <PropTable title="Checkbox" propList={checkboxProps} {...props} />
   </>
 );
+
+export const CheckboxDescriptionPropTable: React.FC<PropTableWrapper> = props => (
+  <PropTable
+    title="CheckboxDescription"
```

Comment:
Should have caught this in the `Radio` update, but can we change these prop table `title`'s to match our other prop tables like `Message[MessageItem]`? This one would be `Checkbox[CheckboxDescription]`.

Reply:
I followed also what it was already there. We currently have `CheckboxLabel`, not `Checkbox[CheckboxLabel]`

I wonder if the "parent wrapping" adds any value